### PR TITLE
birdc, fastd: remove double colon; babeld, bird, birdc, fastd, pystun3, stun: add German translation

### PR DESCRIPTION
**Pull request by marek22k**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/bird.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+> Routing-Daemon mit Unterstützung für BGP, OSPF, Babel und weitere.
+> Weitere Informationen: <https://bird.network.cz/>.
+
+- Starte Bird mit einer bestimmten Konfigurationsdatei:
+
+`bird -c {{path/to/bird.conf}}`
+
+- Starte Bird als spezifischer Benutzer und Gruppe:
+
+`bird -u {{username}} -g {{group}}`
```
Comment (kbdharun) [inline pages.de/common/bird.md:13]:
```suggestion
`bird -u {{benutzername}} -g {{group}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, Can you translate the placeholders too (wherever possible)?

File: pages.de/common/bird.md (hunk 1 above)
Comment (marek22k) [reply pages.de/common/bird.md:13]: Should the arguments all be lowercase or, depending on the context, uppercase? For example, I would capitalize "username" and "group".

Comment (kbdharun) [reply pages.de/common/bird.md:13]: Not sure tbh for German pafe, will ping @pixelcmtd for further remarks

Comment (pixelcmtd) [reply pages.de/common/bird.md:13]: AFAIK, we never capitalize any placeholder, except of course for concrete examples that might have to be capitalized etc

**Review by kbdharun — APPROVED** [review]

File: pages.de/common/stun.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# stun
+
+> Classic STUN client.
```
Comment (pixelcmtd) [inline pages.de/common/stun.md:3]:
```suggestion
> Klassischer STUN-Client.
```

File: pages.de/common/pystun3.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pystun3
+
+> Classic STUN client, geschrieben in Python.
```
Comment (pixelcmtd) [inline pages.de/common/pystun3.md:3]:
```suggestion
> Klassischer STUN-Client, geschrieben in Python.
```

File: pages.de/common/babeld.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# babeld
+
+> Routing-Daemon für Babel, der Firewall-ähnliche Filter benutzt.
+> Weitere Informationen: <https://www.irif.fr/~jch/software/babel/babeld.html>.
+
+- Starte babeld von einer bestimmten Konfigurationsdatei:
```
Comment (pixelcmtd) [inline pages.de/common/babeld.md:6]:
```suggestion
- Starte `babeld` von einer bestimmten Konfigurationsdatei:
```

File: pages.de/common/babeld.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`babeld -c {{pfad/zu/babeld.conf}}`
+
+- Starte babeld von mehreren Konfigurationsdateien (in der Reihenfolge des Einlesens):
```
Comment (pixelcmtd) [inline pages.de/common/babeld.md:10]:
```suggestion
- Starte `babeld` mit mehreren Konfigurationsdateien (in der Reihenfolge des Einlesens):
```

File: pages.de/common/birdc.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+> Bird remote control.
+> Kommandozeilenwerkzeug zum Abrufen von Informationen wie Routen von bird und zur Durchführung von Konfigurationen während der Laufzeit.
+> Weitere Informationen: <https://bird.network.cz/>.
+
+- Öffne die remote control Konsole:
+
+`birdc`
+
+- Lade die Konfiguration neu, ohne Bird neu zu starten.:
```
Comment (pixelcmtd) [inline pages.de/common/birdc.md:11]:
```suggestion
- Lade die Konfiguration neu, ohne Bird neu zu starten:
```

File: pages.de/common/birdc.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Zeige alle Routen an, die eine bestimmte AS-Nummer enthalten:
+
+`birdc "show route where bgp_path ~ [{{4242120045}}]"`
+
+- Zeige alle besten Routen an:
+
+`birdc show route primary`
+
+- Zeige alle Details zu allen Routen von einem bestimmten Präfix anzeigen:
```
Comment (pixelcmtd) [inline pages.de/common/birdc.md:35]:
```suggestion
- Zeige alle Details zu allen Routen von einem bestimmten Präfix an:
```

File: pages.de/common/fastd.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# fastd
+
+> VPN daemon.
+> Arbeitet auf Schicht 2 oder Schicht 3, unterstützt verschiedene Verschlüsselungsmethoden, wird von Freifunk verwendet.
+> Weitere Informationen: <https://fastd.readthedocs.io/en/stable/>.
+
+- fastd von einer bestimmten Konfigurationsdatei starten:
```
Comment (pixelcmtd) [inline pages.de/common/fastd.md:7]:
```suggestion
- Starte `fastd` mit einer bestimmten Konfigurationsdatei:
```

File: pages.de/common/stun.md (hunk 2 above)
Comment (marek22k) [reply pages.de/common/stun.md:3]: Classic here refers to the first RFC standard. Should we translate this anyway?

File: pages.de/common/babeld.md (hunk 4 above)
Comment (marek22k) [reply pages.de/common/babeld.md:6]: Why should you do it this way? After all, this is not the case in the English version either. Furthermore, I find it a bit strange to write proper names or program names in code blocks.

File: pages.de/common/stun.md (hunk 2 above)
Comment (pixelcmtd) [reply pages.de/common/stun.md]: Oh, ok. Then we shouldn't translate it.

File: pages.de/common/babeld.md (hunk 4 above)
Comment (pixelcmtd) [reply pages.de/common/babeld.md]: > this is not the case in the English version either

[Wdym?](https://github.com/tldr-pages/tldr/blob/main/pages/common/babeld.md?plain=1#L6)

> I find it a bit strange to write proper names or program names in code blocks

Well, I don't think we do it with proper names anywhere, but for program names it usually makes sense

Comment (marek22k) [reply pages.de/common/babeld.md]: Mhh, I must have overlooked a few. Thanks for your patience and your good suggestions!

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**